tests/xtimer_reset/tests/01-run.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect_exact("This test tests re-setting of an already active timer.")
+    child.expect_exact("It should print three times \"now=<value>\", with "
+                       "values approximately 100ms (100000us) apart.")
+    child.expect(u"now=\d+")
```

Comment:
When setting the `\d+` in parenthesis you can check value actually with regards to the statement above by using `child.match.group(1)`. `child.match` is a [Match Object](https://docs.python.org/3/library/re.html#match-objects).

Reply:
(this is more of a suggestion and not blocking this PR)

Reply:
> `child.match` is a [Match Object]

(this also means, that `child.match.group(1)` will be a string, but since you already assumed it to be a number with `\d+` you can safely convert it to `int`.

Reply:
Yep, but the question of choosing the correct passing ranges/intervals/thresholds is not quite trivial. Thats why I would like to merge this PR as is for now as first step, and take the interpretation of the concrete numbers into (a) follow up(s).

Reply:
Mhhh... I think that at least the fact that the number is growing should be easily testable, but okay: let's move on for now.